akka-docs/src/main/paradox/typed/logging.md

```diff
@@ -173,9 +173,9 @@ in the `ActorContext`.
 
 ## SLF4J API compatibility
 
-The SLF4J API broke binary compatibility between versions 1.7 and 2.0. Akka depends on 1.7 but supports either versions. 
+Since Akka 2.10.0, only SLF4j version `2.0` is supported. 
 
-It is however not possible to mix a logger backend supporting one version with SLF4J API of other version, that will lead
+It is not possible to mix a logger backend supporting one version with SLF4J API of older versions, that will lead
 to no logging to output like this:  
```

Comment:
Is below log message obsolete now? I don't think we want to encourage downgrading slf4j as is explained in below workaround.

Reply:
Yes, lets remove all the "working around this" docs.